### ISSUE TITLE
FAIL_ON_NULL_FOR_PRIMITIVES not working with AfterburnerModule

### ISSUE THREAD

**Issue by Qnzvna** [FasterXML/jackson-modules-base#35]
Hi,
I've tried to used AfterburnerModule with Jackson DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES but it look like it doesn't work for long and int primitives. Double works fine.

Here is the code scratch to reproduce the problem.

```
class Test
{
    public static void main(String[] args)
    {
        final ObjectMapper mapper = new ObjectMapper()
                .registerModule(new AfterburnerModule())
                .enable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES);
        try {
            final LongWrapper longWrapper = mapper.readValue("{\"value\": null}", LongWrapper.class);
            System.out.println(longWrapper);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }

    static class LongWrapper
    {
        public long value;
    }
}
```

I'm using Jackson 2.9.1 with Dropwizard 1.2.0.

**Comment by Qnzvna** [comment]
I've opened the PR #36

**Comment by cowtowncoder** [comment]
Fixed via #38, which is listed in release notes.
